CMakeLists.txt

```diff
@@ -79,14 +81,15 @@ add_custom_target(header_check ALL
 	${CMAKE_COMMAND} "-Domr_SOURCE_DIR=${omr_SOURCE_DIR}" -P ${CMAKE_CURRENT_SOURCE_DIR}/cmake/CheckSourceTree.cmake)
 endif()
 
-configure_file(./omrcfg.CMakeTemplate.h omrcfg.h)
-configure_file(./omrversionstrings.CMakeTemplate.h omrversionstrings.h)
+configure_file(./omrcfg.CMakeTemplate.h ${CMAKE_BINARY_DIR}/omrcfg.h)
+configure_file(./omrversionstrings.CMakeTemplate.h ${CMAKE_BINARY_DIR}/omrversionstrings.h)
```

Comment:
This was a side effect of an earlier experiment. I'm just going to drop this patch.